lib/src/constants/constant_text.dart

```diff
@@ -55,3 +55,38 @@ class ForgotPasswordText {
   static const String sendButton = "Send";
   static const String tabletLabel = "Start from Scratch";
 }
+
+class NewRecipeText {
+  static const String hintRecipeNameText = "Write Down Recipe Name";
+  static const String labelGalleryText = "Gallery";
+  static const String labelIngredientsText = "Ingredients";
+  static const String labelHowToCookText = "How to Cook";
+  static const String labelAdditionalInfoText = "Additional Info";
+  static const String hintGalleryText = "Upload Images or Open Camera";
+  static const String hintIngredientsText = "Add Ingredient";
+  static const String hintHowToCookText = "Add Directions";
+  static const String hintAdditionalInfoText = "Add Info";
+  static const String saveToText = "Save to";
+  static const String saveRecipeText = "Save Recipe";
+  static const String labelRecipeNameText = "Recipe Name";
+  static const String postToFeedText = "Post to Feed";
+  static const String leadingText = "Back to My Recipes";
+  static const String titleText = "New Recipe";
+  static const String titleTabletText = "Create New Recipe";
+  static const String camera = "Camera";
+  static const String gallery = "Gallery";
+  static const String chooseOption = "Choose option";
+  static const String stepHintText = "Add step";
+  static const String saveInfoText = "Save info";
+  static const String tagsText = "Tags";
+  static const String nutritionFactText = "Nutrition Fact";
+  static const String servingTimeText = "Serving Time (±)";
+  static const String saveNewRecipeSuccessText =
+      'The recipe was saved successfully';
+  static const String saveNewRecipeFailureText = 'Oops! Something wrong';
+  static const String AddNewCategoryText = '+ New category';
```

Comment:
do you need to change the name k "addNewCategoryText"

Reply:
Thanks I will fix it right away.